dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GitHubWorkflowIT.java

```diff
@@ -194,6 +196,29 @@ void testPublishingAndListingOfPublished() throws ApiException {
             WorkflowSubClass.BIOWORKFLOW.getValue()).size(), "Published workflow count should be unchanged");
     }
 
+    /**
+     * Tests that the correct error is given when provided an invalid value for sortCol when getting all published workflows
+     *
+     */
+    @Test
+    void testGetPublishedWorkflowsWithInvalidSortCol() {
+        final ApiClient webClient = getWebClient(USER_2_USERNAME, testingPostgres);
+        WorkflowsApi workflowApi = new WorkflowsApi(webClient);
+        final PublishRequest publishRequest = CommonTestUtilities.createPublishRequest(true);
+
+        AppToolHelper.registerAppTool(webClient);
+        Workflow appTool = workflowApi.getWorkflowByPath("github.com/" + toolAndWorkflowRepoToolPath, APPTOOL, "versions");
+        workflowApi.publish(appTool.getId(), publishRequest);
+
+        assertEquals(1, workflowApi.allPublishedWorkflows(null, null, null, null, null, false,
+                WorkflowSubClass.APPTOOL.getValue()).size(), "There should be 1 app tool published");
+
+        ApiException exception = assertThrows(ApiException.class, () -> workflowApi.allPublishedWorkflows(null, null, null, "invalid", null, false,
+                    WorkflowSubClass.APPTOOL.getValue()));
+        assertTrue(exception.getMessage().contains("Could not process query due to the invalid sortCol value."));
```

Comment:
Use a shared String constant for this and in the class itself `EntryDAO`, that way the test will automatically keep up-to-date if the message changes